### ISSUE TITLE
Registration form requests not going through

### ISSUE THREAD

**Issue by artfuldev** [artfuldev/devday#57]
CORS or some other issue, the first OPTIONS pre-flight request itself is failing
```
Request URL:https://docs.google.com/a/sahajsoft.com/forms/d/e/1FAIpQLSd7wUzgQ7VuP3z41GtnTemaxFzv-4K10TuBHjCZqjcI8xxDJA/formResponse
Request Method:OPTIONS
Status Code:405 
Remote Address:216.58.197.46:443
Response Headers
alt-svc:quic=":443"; ma=2592000; v="35,34,33,32,31,30"
alternate-protocol:443:quic
cache-control:private, max-age=0
content-encoding:gzip
content-length:127
content-type:text/html; charset=UTF-8
date:Mon, 29 Aug 2016 04:52:00 GMT
expires:Mon, 29 Aug 2016 04:52:00 GMT
server:GSE
status:405
x-content-type-options:nosniff
x-frame-options:SAMEORIGIN
x-xss-protection:1; mode=block
Request Headers
Provisional headers are shown
Access-Control-Request-Headers:content-type
Access-Control-Request-Method:POST
Origin:https://artfuldev.github.io
Referer:https://artfuldev.github.io/devday/
User-Agent:Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/52.0.2743.116 Safari/537.36
```

**Comment by artfuldev** [comment]
The place to edit is `src\drivers\registration.ts`

**Comment by artfuldev** [comment]
started work in https://github.com/artfuldev/devday/tree/fix-reg-form-requests

**Comment by artfuldev** [comment]
Fixed in #67
